README.md

```diff
@@ -1,211 +1,20 @@
 # ros2_control
 
-## CI
-linux: [![Build Status](https://travis-ci.org/ros-controls/ros2_control.svg?branch=master)](https://travis-ci.org/ros-controls/ros2_control)
+[![Licence](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Disclaimer
+|         | Foxy    | Rolling
+|:-------:|:-------:|:-------:|
+| Branch  | [`master`](https://github.com/ros-controls/ros2_control/tree/master) | TBD
+| CI  | Linux: [![Build Status](https://github.com/ros-controls/ros2_control/workflows/Test%20ros2_control/badge.svg)](https://github.com/ros-controls/ros2_control/actions?query=workflow%3A"Test%20ros2_control")| TBD |
+| Linters  | [![Linters Status](https://github.com/ros-controls/ros2_control/workflows/Lint%20ros2_control/badge.svg)](https://github.com/ros-controls/ros2_control/actions?query=workflow%3A"Lint+ros2_control")| TBD |
 
-`ros2_control` is under heavy development, to get an update of the project please either tag along to one of our working group meetings (see more on [discourse](https://discourse.ros.org/)) and/or check the resources below:
-* Actively tracked projects can be found here: https://github.com/orgs/ros-controls/projects
-* Consult the roadmap here: https://github.com/ros-controls/roadmap
-* See demos here (under heavy development): https://github.com/ros-controls/ros2_control_demos
 
-`ros2_control` is a proof of concept on how new features within ROS 2 can be elaborated and used in the context of robot control.
-We were keeping the vocabulary close the ROS 1 implemenation, however this is with no notion a migration, but rather a fresh new-write.
-We hope that this can be a starting point for migrating ros_control to ROS 2 at some point.
-Reasons for re-writing this repo from scratch rather than porting existing code to ROS 2 syntax is to leverage the new ROS 2 concepts in full.
-The work done in this repo (together with [`ros2_controllers`](https://github.com/ros-controls/ros2_controllers)) was presented at [ROSCon 2017](https://vimeo.com/236182180) and is currently not under active development.
+## About
+This package is a part of the ros2_control framework. 
+For further description check the [documentation](https://github.com/ros-controls/control.ros.org).
```

Comment:
Shouldn't it point to https://ros-controls.github.io/control.ros.org/ and later to https://control.ros.org/?

If someones is reading the readme, probably they just want the doc, not the source files for the doc.

Reply:
```suggestion
For further description check the [documentation](https://ros-controls.github.io/control.ros.org/).
```